<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>373</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cyrus Visualizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*.QGroupBox {background-color : darkgreen}*/
/*QWidget {background-color : green}
QPushButton{
	background-color : black;
	color : yellow;
	radius : 3px
}*/
QWidget {
font: 9px &quot;Ubuntu&quot;
}

QTabWidget::pane { /* The tab widget frame */
     border-top: 0px solid #C2C7CB;
 }

QTabWidget::tab-bar {
     left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     border-top-left-radius: 1px;
     border-top-right-radius: 1px;
     min-width: 8ex;
     padding: 1px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 }

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }

 QTabBar::tab:!selected {
     margin-top: 0px; /* make non-selected tabs look smaller */
 }

QStatusBar QLabel {
    border: 2px solid magenta;
	font: bold 14px;
}
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="leftLayout" stretch="0,0,1,0,0">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QGroupBox" name="scoreBoard">
          <property name="title">
           <string>Game State</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="gameRunningLabel">
             <property name="text">
              <string>Game Running</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="refereeLabel">
             <property name="text">
              <string>Referee</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="strategyLabel">
             <property name="text">
              <string>Strategy</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>OurColor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="ourRobotsTableWidget">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>60</number>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Ball</string>
           </property>
           <item>
            <property name="text">
             <string>position</string>
            </property>
            <item>
             <property name="text">
              <string>x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>y</string>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>speed</string>
            </property>
            <item>
             <property name="text">
              <string>speed_x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>speed_y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>speed_mag</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>speed_teta</string>
             </property>
            </item>
           </item>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_1" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="fieldWidget" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="fieldWidgetLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="plotterContainerWidget" native="true">
         <layout class="QVBoxLayout" name="plotManagerLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="scatterContainerWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>53</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="scatterLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFull_Screen"/>
   <addaction name="actionConnect_Server"/>
   <addaction name="actionShow_Plans"/>
   <addaction name="actionShow_Targets"/>
   <addaction name="actionShowIntersects"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect_Server">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/network.png</normaloff>:/icons/network.png</iconset>
   </property>
   <property name="text">
    <string>Connect Server</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/fullscreen.png</normaloff>:/icons/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionShow_Plans">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/show-plan.png</normaloff>:/icons/show-plan.png</iconset>
   </property>
   <property name="text">
    <string>Show Plans</string>
   </property>
  </action>
  <action name="actionShow_Targets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/show-target.png</normaloff>:/icons/show-target.png</iconset>
   </property>
   <property name="text">
    <string>Show Targets</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShowIntersects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Soccer_ball.svg.png</normaloff>:/icons/Soccer_ball.svg.png</iconset>
   </property>
   <property name="text">
    <string>showIntersects</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
